frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -84,7 +84,11 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
               </NavDropdown>
             </Nav>
 
-
+            <Nav className="mr-auto">
+              <NavDropdown title="Course Instructor" id="appnavbar-course-instructor-dropdown" data-testid="appnavbar-course-instructor-dropdown" >
+                <NavDropdown.Item href="/courseinstructor/search" data-testid="appnavbar-course-instructor-search">Search</NavDropdown.Item>
+              </NavDropdown>
+            </Nav>
```

Comment:
If a dropdown has only one item in it, why even make it a drop down?  Why not just make it a link?

See https://react-bootstrap.github.io/docs/components/navbar for options and examples.

Also: see my feedback on #29 ... I'm just not sold on "Course Instructor" as a helpful label for this activity.  Maybe you can convince me, but I think "search by instructor" or even "Search by" with instructor / location / course number as the things in the dropdown menu would make much more sense...